app/controllers/v1/events_controller.rb

```diff
@@ -0,0 +1,25 @@
+module V1
+  class EventsController < ApplicationController
+    before_action :authenticate_user, except: [:show, :index]
+
+    def create
+      existing_event = Event.where(event_lookup_params).first
+
+      if existing_event
+        redirect_to(action: :show, id: existing_event.id)
+      else
+        super
+      end
+
+    end
+
+    private
+
+      def event_lookup_params
```

Comment:
I've seen it go both ways, I honestly don't know that there's a one true rule on this. I can bump them to the left if that's a norm or preference for the codebases.

My rationale is that the extra indent helps me know at a glance that I'm in a private/protected section of a file.